ja-jp/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm

```diff
@@ -311,7 +311,7 @@ <h4><a name="User"></a>OAuth 2.0とOpenID Connectのフローに対するユー
                                                     <p> 新規のアクセストークンが必要な場合、アプリケーションは、ユーザーが特定のスコープに同意済みであれば、同意を求める必要はありません。承諾は、ユーザーが手動で失効させるか、ユーザー、統合サーバー、認可サーバー、またはスコープが非アクティブ化または削除されるまで有効です。ユーザーの同意が機能するには、orgの<a href="../Security/API_Access.htm">［API Access Management（APIアクセス管理）］</a>機能が有効化されている必要があります。</p>
                                                     <h2><a name="Task"></a>タスク1：ウィザードの起動</h2>
                                                     <ol>
-                                                        <li value="1"><span class="mc-variable okta-feature-names.Administrator_dashboard variable">Okta Admin Console</span>で、<span class="uicontrol">［Applications（アプリケーション）］</span>&gt;<span class="uicontrol">［Applications（アプリケーション）］</span>に移動します。</li>
```

Comment:
Same as in OIE, "Okta Admin Console" should not be translated since it is in the DNT. This should be fixed next round